hr_timesheet_timetracker/models/account_analytic_line.py

```diff
@@ -0,0 +1,363 @@
+# Copyright 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+from datetime import timedelta
+
+from odoo import fields, models, api, _
+from odoo.exceptions import UserError, ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = 'account.analytic.line'
+    _order = (
+        'date desc, create_date desc, id desc'
+    )
+
+    create_date = fields.Datetime(
+        index=True,
+    )
+    is_timetracker_running = fields.Boolean(
+        string='Timetracker Running',
+        compute='_compute_is_timetracker_running',
+        store=True,
+    )
+    is_timetracked = fields.Boolean(
+        string='Timetracked Line',
+        compute='_compute_is_timetracked',
+        store=True,
+    )
+    timetracker_started_at = fields.Datetime(
+        string='Started At',
+        copy=False,
+        index=True,
+        help='Timestamp when timetracker was started',
+    )
+    timetracker_stopped_at = fields.Datetime(
+        string='Stopped At',
+        copy=False,
+        index=True,
+        help='Timestamp when timetracker was stopped',
+    )
+    can_use_timetracker = fields.Boolean(
+        string='Can Track Time',
+        compute='_compute_can_use_timetracker',
+        default=True,
+    )
+
+    _sql_constraints = [
+        (
+            'single_running_timetracker_per_employee',
+            (
+                'EXCLUDE (employee_id WITH =) WHERE ('
+                '    timetracker_started_at IS NOT NULL'
+                '    AND timetracker_stopped_at IS NULL'
+                ')'
+            ),
+            'Only one running timetracker allowed per employee'
+        ),
+    ]
+
+    @api.depends('timetracker_started_at', 'timetracker_stopped_at')
+    def _compute_is_timetracker_running(self):
+        for line in self:
```

Comment:
@rvalyi In loop add `line.is_timetracked = (True if line.timetracker_started_at else False)` 

It's not need to second [_compute_is_timetracked]( https://github.com/OCA/hr-timesheet/pull/139/files#diff-dbf0743454016a436eaa3a48eca07921R71) here

Reply:
Yet why to trigger that if only `timetracker_stopped_at` changed?

Reply:
> Yet why to trigger that if only `timetracker_stopped_at` changed?

@alexey-pelykh here one time call depends `timetracker_started_at` then why you recall other method.

Reply:
@nikul-serpentcs true, yet I'm not sure that `is_timetracked` needs to be recomputed (and thus rewritten) on `timetracker_stopped_at` change. Is there a guideline for that?

Reply:
> @nikul-serpentcs true, yet I'm not sure that `is_timetracked` needs to be recomputed (and thus rewritten) on `timetracker_stopped_at` change. Is there a guideline for that?

@alexey-pelykh No, I have not any guideline for that.

If call `_compute_is_timetracker_running` then depends on (`'timetracker_started_at', 'timetracker_stopped_at'`)

If call `_compute_is_timetracked` then depends on only this (`'timetracker_started_at'`)

If change `timetracker_started_at` then call two compute method right?

That's why, only call one method in two compute.

Reply:
On other hand, if `timetracker_stopped_at` only one field get's updated, as it should be. Seems to me that calling single method instead of calling two methods and updating two fields when it's only one field that had to be updated is likewise not 100% ok.

It's more logical (IMHO) to keep two methods and avoid unnecessary field update

Reply:
@alexey-pelykh OK

Reply:
thanks for constructive feedback!